### PR TITLE
New Metric Schema

### PR DESCRIPTION
**Pull request by tiffanyfj**

### CODE REVIEW

File: dbi/dbi.go [hunk 1]
```diff
@@ -76,14 +77,11 @@ func (dbiPlg *DbiPlugin) CollectMetrics(mts []plugin.PluginMetricType) ([]plugin
...
-		if value, ok := data[joinNamespace(m.Namespace())]; ok {
-			metric := plugin.PluginMetricType{
+		if value, ok := data[m.Namespace().String()]; ok {
+			metric := plugin.MetricType{
 				Namespace_: m.Namespace(),
 				Data_:      value,
-				Source_:    hostname,
 				Timestamp_: time.Now(),
 				Version_:   m.Version(),
 			}
```
Comment (IzabellaRaulin) [inline dbi/dbi.go]: I have a question what about tags - is there should added also: `Tags_: m.Tags()` or not. What is the best approach?

**Comment by IzabellaRaulin** [comment]
For this snap commit the global config has no more: ` "cache_ttl": "5s"`, "default_deadline": "5s", and `"worker_pool_size": 5` - could You also remove it from examplary configuration files? 

To keep consistency some updates in README.md (in Examples section) might be needed which refers to changes in global config examples and increment version of dbi plugin from 3 to 4.

Others that that, LGTM

**Comment by tiffanyfj** [comment]
Thanks @IzabellaRaulin :)

**Comment by IRCody** [comment]
LGTM

**Pull request merged**